auditbeat/docs/modules_list.asciidoc

```diff
@@ -8,5 +8,6 @@ This file is generated! See scripts/docs_collector.py
 
 --
 
+include::./modules/.DS_Store.asciidoc[]
```

Comment:
@adriansr @cwurm This probably needs some fixing in the collection script. Will fix it in this PR manually.